docs/devel/networking.md

```diff
@@ -326,3 +326,51 @@ Chain POSTROUTING (policy ACCEPT)
 target     prot opt source               destination
 MASQUERADE  all  --  10.0.2.2             anywhere
 ```
+
+### Masquerade binding using IPv6 addresses
+The masquerade binding mechanism is currently the only binding mechanism which
+accepts IPv6 addresses.
+
+It operates in exactly the same way as in IPv4, and follows the same goals:
+configure one to one NAT. As in it's IPv4 counter-part, the pods are reached
+via their IPv6 pod addresses.
+
+NAT is configured in the exact same way, but using ip6tables, or the `ipv6-nat`
+nftable table. Please refer to the tables below to visualize how NAT for IPv6
+addresses is accomplished in KubeVirt.
+
+```
+Chain PREROUTING (policy ACCEPT)
+target               prot opt source               destination
+KUBEVIRT_PREINBOUND  all      anywhere             anywhere
+
+Chain INPUT (policy ACCEPT)
+target     prot opt source               destination
+
+Chain OUTPUT (policy ACCEPT)
+target     prot opt source               destination
+DNAT       tcp      anywhere             localhost            tcp dpt:http to:fd10:0:2::2
+
+Chain POSTROUTING (policy ACCEPT)
+target     prot opt source               destination
+MASQUERADE  all      fd10:0:2::2          anywhere
+KUBEVIRT_POSTINBOUND  all      anywhere   anywhere
+
+Chain KUBEVIRT_POSTINBOUND (1 references)
+target     prot opt source               destination
+SNAT       tcp      anywhere             anywhere             tcp dpt:http to:fd10:0:2::1
+
+Chain KUBEVIRT_PREINBOUND (1 references)
+target     prot opt source               destination
+DNAT       tcp      anywhere             anywhere             tcp dpt:http to:fd10:0:2::2
+```
+
+It is important to refer that when IPv6 masquerade binding is set in place,
```

Comment:
The example of the iptables above is for a vm with no specific ports. I would mention that is case ports are specified for the vm, only traffic to those specific ports will be forwarded to the vm.

Reply:
Hm, for [ipv4 masquerade that has already been explained](https://github.com/kubevirt/kubevirt/commit/1d2295fcd9804db0f4965c3900d41934259bec84#diff-971521d7f1fea6eeaa462181948819a9R231-R234). Nothing new here.

I can provide an example w/ ports defined if you prefer. What do you think ?

Reply:
If you have one with port for ipv4 it is enough.

Reply:
I do:
  - https://github.com/kubevirt/kubevirt/commit/1d2295fcd9804db0f4965c3900d41934259bec84#diff-971521d7f1fea6eeaa462181948819a9R311
  - https://github.com/kubevirt/kubevirt/commit/1d2295fcd9804db0f4965c3900d41934259bec84#diff-971521d7f1fea6eeaa462181948819a9R291